locales/en.default.schema.json

```diff
@@ -456,6 +456,20 @@
         },
         "show_social": {
           "label": "Show icons on desktop"
+        },
+        "header__3": {
+          "content": "Country/region selector",
+          "info": "To add a country/region, go to your [market settings.](/admin/settings/markets)"
```

Comment:
Also not worth changing with translations already done, but I'd probably keep the period out of the link markdown.